mods/ra/rules/infantry.yaml

```diff
@@ -328,6 +342,11 @@ SPY:
 		DefaultAttackSequence: shoot
 		IdleSequences: idle1,idle2
 		StandSequences: stand,stand2
+		RequiresCondition: !parachute
+	WithDisguisingInfantryBody@PARACHUTE:
+		RequiresCondition: parachute
+		Palette: player-noshadow
+		IsPlayerPalette: true
```

Comment:
This drops the `stand2` sequence for parachuting, but not sure how relevant that is while dropping.

Reply:
This was intentional on my end, I felt for parachuting consistency with other infantry would be more important.